<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanMainGUI</class>
 <widget class="QWidget" name="ScanMainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScanMainGUI</string>
  </property>
  <property name="toolTip">
   <string>确认选择</string>
  </property>
  <widget class="QPushButton" name="leftViewBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>leftViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="frontviewBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>前视</string>
   </property>
   <property name="text">
    <string>frontview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modelMoveStateSetBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>40</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>框选</string>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>selectRegion</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="jawViewBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>颌视</string>
   </property>
   <property name="text">
    <string>jawViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightViewBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>右视</string>
   </property>
   <property name="text">
    <string>rightViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backViewBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>后视</string>
   </property>
   <property name="text">
    <string>backViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ChangeColor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="narrowViewBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="text">
    <string>narrowViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enlargeViewBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="text">
    <string>enlargeViewBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delSelectedBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>框选</string>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>delSelected</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmSelectedBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>确认选择</string>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="text">
    <string>confirmSelected</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bgGroundmoveUpBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>bgGroundmoveUpBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bgGroundmoveDownBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>bgGroundmoveDownBtn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bgGroundShowBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>bgGroundShowBtn</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cutModelUnderBgBtn">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>90</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>cutModelUnderBgBtn</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditR">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>80</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditG">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>80</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditB">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>80</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeBgColorBtn">
   <property name="geometry">
    <rect>
     <x>1770</x>
     <y>100</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>左视</string>
   </property>
   <property name="text">
    <string>changeBgColorBtn</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditA">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>80</y>
     <width>104</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="ScanJawGroup">
   <property name="geometry">
    <rect>
     <x>1630</x>
     <y>230</y>
     <width>281</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="ScanJawTips">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请插入下颌模型</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScanJawGroupTipImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/images/alljaw.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ScanJawScanBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ScanJawNextStepBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一步</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
